### PR TITLE
Windows/Mac setup guides with simple tutorial and screenshots

### PR DESCRIPTION
**Pull request by eggplantparm**
### Description

Hello! I became a phpMyAdmin user earlier this year while pursuing a computer science degree. A class project required some sort of contribution to a project we're interested in. I tried out some newbie bug fixes from triage, but I was not confident in my fixes, so I looked through the doc directory. I think that the Windows section for phpMyAdmin could expand a bit for beginners, so I added some newbie guides for Windows and Mac OS X users to the doc directory. This may be suited for a wiki contribution instead of repo contribution?

Based on previous documentation pull requests below, I'm not sure if you agree that the community needs further documentation, even for the Wiki page... but, approved or not, thank you all for keeping this community thriving over the years. I look forward to using the software and providing code contributions - not just doc contributions - in the future. :)

Related to #16460 (2021), #16200 (2021), #16161 (2020), #15964 (2020), #15942 (2020)
Added: "getting_started" directory containing .md and .rst beginner tutorials for phpMyAdmin, and .md and .rst guides for Windows or Mac OS X setup
Added: "images" sub-directory for tutorial guidance
Modified: setup.rst to include redirect to Windows and Mac OS X guides
Branch based on QA_5_2

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by ibennetch** [comment]
This is quite comprehensive — thank you for putting in the work on this.

It may be that this is better suited for the wiki, but there's a lot to go through for me to have an opinion about that. So I'll take a bit to look it over in the meantime. But thanks!

File: doc/getting_started/osx_setup.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# OS X Setup Guide
```
Comment (williamdes) [inline doc/getting_started/osx_setup.md:1]: I am not sure .md files are useful as they mostly contain HTML. And HTML is generated using .rst files and distributed online and as a local documentation

**Comment by williamdes** [comment]
> It may be that this is better suited for the wiki, but there's a lot to go through for me to have an opinion about that.

I also agree with this. I think this is very nice to have such tutorial. The wiki pages will be good for that: https://github.com/phpmyadmin/phpmyadmin/wiki

I think they are markdown only, but not sure. Anyway you have two working and good formats :)
Let's wait for @ibennetch but I think you can publish all your work on the wiki

**Comment by eggplantparm** [comment]
Thank you both for the kind feedback. I agree - I'll work on getting some markdown guides suitable for the wiki. At the moment, I think we can go ahead and close the request to the repository, would that be OK?

**Comment by williamdes** [comment]
Yes, definitelywe can use this PR to discuss the wiki changes

**Pull request closed without merging**